src/Magento/FunctionalTestingFramework/Util/ModuleResolver.php

```diff
@@ -201,7 +202,7 @@ public function getEnabledModules()
 
         $token = $this->getAdminToken();
 
-        $url = ConfigSanitizerUtil::sanitizeUrl(getenv('MAGENTO_BASE_URL')) . $this->moduleUrl;
+        $url = UrlFormatter::format(getenv('MAGENTO_BASE_URL')) . $this->moduleUrl;
```

Comment:
Instead of a generic format() function, I expected the new class to operate like:

`AwesomeNewClass::getMagentoBaseUrl()`
`AwesomeNewClass::getBackendUrl()`

Maybe I was thinking wrong? What do you think?

Reply:
In the refactored code, you can see that we have a few places that you can query magentoBaseUrl() or backendUrl(). 
This interface util is to standardize the use of user specified variable regardless of trailing /, etc. If you look at the various refactored code, you can see the vulnerability of the way we use user specified envs, e.g. TESTS_MODULE_PATH, TESTS_BP, etc

Reply:
Gotcha.